050-AIS-APIManagementwithFunctions/Coach/README.md

```diff
@@ -0,0 +1,85 @@
+# What The Hack - AIS - API Management with Function Apps - Coach Guide
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+
+## Introduction
+
+<!-- REMOVE_ME Welcome to the coach's guide for the ${nameOfChallengeArg} What The Hack. Here you will find links to specific guidance for coaches for each of the challenges. (remove this from your MD files if you are writing them manually, this is for the automation script) REMOVE_ME -->
+
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+Welcome to the coach's guide for the Azure Integration Services - API Management with Function Apps Hack. Here you will find links to specific guidance for coaches for each of the challenges.
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+
+<!-- This hack includes an optional [lecture presentation](Lectures.pptx) that features short presentations to introduce key topics associated with each challenge. It is recommended that the host present each short presentation before attendees kick off that challenge. -->
+
+**NOTE:** If you are a Hackathon participant, this is the answer guide. Don't cheat yourself by looking at these during the hack! Go learn something. :)
+
+## Coach's Guides
+<!-- REMOVE_ME ${challengesSection} (remove this from your MD files if you are writing them manually, this is for the automation script) REMOVE_ME -->
+
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+-  Challenge 0: **[Prepare your Development Environment](Solution-00.md)**
+   - Get yourself ready to build your integration solution
+-  Challenge 1: **[Provision an Integration Environment](Solution-01.md)**
+   - Create a bicep template that will provision a baseline integration environment.
+-  Challenge 2: **[Deploy your Integration Environment](Solution-02.md)**
+   - Create a CI/CD pipeline to do automated deployment of your integration environment.
+-  Challenge 3: **[Create Backend APIs](Solution-03.md)**
+   - Create backend APIs
+-  Challenge 4: **[Secure Backend APIs](Solution-04.md)**
+   - Securing backend APIs by configuring them in the VNET or by using OAuth 2.0 authorization
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+
+## Coach Prerequisites 
+
+This hack has pre-reqs that a coach is responsible for understanding and/or setting up BEFORE hosting an event. Please review the [What The Hack Hosting Guide](https://aka.ms/wthhost) for information on how to host a hack event.
+
+The guide covers the common preparation steps a coach needs to do before any What The Hack event, including how to properly configure Microsoft Teams.
+
+### Student Resources
+
+Before the hack, it is the Coach's responsibility to download and package up the contents of the \`/Student/Resources\` folder of this hack into a "Resources.zip" file. The coach should then provide a copy of the Resources.zip file to all students at the start of the hack.
```

Comment:
Remove the backslashes around the folder path.  They were escape characters needed by our automation script.  Unfortunately, when using the template manually, you need to remove the escape characters.  

Please DO leave the backticks in place so the folder path appears in monospace.

Reply:
Done